slices/tzdata.yaml

```diff
@@ -0,0 +1,300 @@
+      /usr/share/zoneinfo/posix/Africa/*:
+      /usr/share/zoneinfo/right/Africa/*:
+      /usr/share/zoneinfo/Egypt:
+      /usr/share/zoneinfo/Iceland:
+      /usr/share/zoneinfo/Libya:
+      /usr/share/zoneinfo/posix/Egypt:
+      /usr/share/zoneinfo/right/Egypt:
+      /usr/share/zoneinfo/posix/Iceland:
+      /usr/share/zoneinfo/right/Iceland:
+      /usr/share/zoneinfo/posix/Libya:
+      /usr/share/zoneinfo/right/Libya:
+
+  america:
+    contents:
+      /usr/share/zoneinfo/America/Argentina/*:
+      /usr/share/zoneinfo/America/Indiana/*:
+      /usr/share/zoneinfo/America/Kentucky/*:
+      /usr/share/zoneinfo/America/North_Dakota/*:
+      /usr/share/zoneinfo/America/*:
+      /usr/share/zoneinfo/posix/America/*:
+      /usr/share/zoneinfo/posix/America/Argentina/*:
+      /usr/share/zoneinfo/posix/America/Indiana/*:
+      /usr/share/zoneinfo/posix/America/Kentucky/*:
+      /usr/share/zoneinfo/posix/America/North_Dakota/*:
+      /usr/share/zoneinfo/right/America/*:
+      /usr/share/zoneinfo/right/America/Argentina/*:
+      /usr/share/zoneinfo/right/America/Indiana/*:
+      /usr/share/zoneinfo/right/America/Kentucky/*:
+      /usr/share/zoneinfo/right/America/North_Dakota/*:
+      /usr/share/zoneinfo/Cuba:
+      /usr/share/zoneinfo/Jamaica:
+      /usr/share/zoneinfo/Navajo:
+      /usr/share/zoneinfo/posix/Cuba:
+      /usr/share/zoneinfo/right/Cuba:
+      /usr/share/zoneinfo/posix/Jamaica:
+      /usr/share/zoneinfo/right/Jamaica:
+      /usr/share/zoneinfo/posix/Navajo:
+      /usr/share/zoneinfo/right/Navajo:
+      /usr/share/zoneinfo/posixrules:
+
+  antarctica:
+    essential:
+      - tzdata_eurasia
+      - tzdata_pacific
+    contents:
+      /usr/share/zoneinfo/Antarctica/*:
+      /usr/share/zoneinfo/posix/Antarctica/*:
+      /usr/share/zoneinfo/right/Antarctica/*:
+
+  arctic:
+    essential:
+      - tzdata_eurasia
+    contents:
+      /usr/share/zoneinfo/Arctic/*:
+      /usr/share/zoneinfo/posix/Arctic/*:
+      /usr/share/zoneinfo/right/Arctic/*:
+
+  atlantic:
+    essential:
+      - tzdata_africa
+      - tzdata_eurasia
+    contents:
+      /usr/share/zoneinfo/Atlantic/*:
+      /usr/share/zoneinfo/posix/Atlantic/*:
+      /usr/share/zoneinfo/right/Atlantic/*:
+
+  australia:
+    contents:
+      /usr/share/zoneinfo/Australia/*:
+      /usr/share/zoneinfo/posix/Australia/*:
+      /usr/share/zoneinfo/right/Australia/*:
+
+  brazil:
+    essential:
+      - tzdata_america
+    contents:
+      /usr/share/zoneinfo/Brazil/*:
+      /usr/share/zoneinfo/posix/Brazil/*:
+      /usr/share/zoneinfo/right/Brazil/*:
+
+  canada:
+    essential:
+      - tzdata_america
+    contents:
+      /usr/share/zoneinfo/Canada/*:
+      /usr/share/zoneinfo/posix/Canada/*:
+      /usr/share/zoneinfo/right/Canada/*:
+
+  chile:
+    essential:
+      - tzdata_america
+      - tzdata_pacific
+    contents:
+      /usr/share/zoneinfo/Chile/*:
+      /usr/share/zoneinfo/posix/Chile/*:
+      /usr/share/zoneinfo/right/Chile/*:
```

Comment:
As `canada`, I would put `chile` in `america`.

Reply:
(same as https://github.com/canonical/chisel-releases/pull/16/files#r1297030775)